tests/detect-bypass/test.rules

```diff
@@ -0,0 +1,3 @@
+alert tcp any any -> any any (bypass; content:"GET "; sid:1;)
```

Comment:
hmm not sure what I'm missing but seems like a valid match to me, don't know the reason why it fails in C..